Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -0,0 +1,761 @@
+    soup = BeautifulSoup(html)
+    for a in soup.findAll('a'):
+        if a.has_attr('href'):
+            if a['href'] not in a.get_text():
+                embedded_urls.append(a['href'])
+    return embedded_urls
+
+
+def get_urls_to_run(email_body, email_html, urls_argument, max_urls, model, msg_list, debug):
+    if email_body:
+        urls_email_body = extract_urls(email_body)
+    else:
+        if (not email_body and email_html):
+            urls_email_body = extract_urls(BeautifulSoup(email_html).get_text())
+        else:
+            urls_email_body = []
+    if email_html:
+        urls_email_html = extract_embedded_urls_from_html(email_html)
+    else:
+        urls_email_html = []
+    if isinstance(urls_argument, list):
+        urls_only = urls_argument
+    else:
+        urls_only = [x.strip() for x in urls_argument.split(' ') if x]
+    urls = urls_email_body + urls_only + urls_email_html
+    if not urls:
+        msg_list.append(MSG_NO_URL_GIVEN)
+        demisto.results(MSG_NO_URL_GIVEN)
+        return
+    urls = get_final_urls(urls, max_urls, model)
+    urls = [demisto.executeCommand("UnEscapeURLs", {"input": x})[0]['Contents'] for x in urls]
+    if debug:
+        demisto.results(urls)
+    return urls, msg_list
+
+
+def update_and_load_model(debug, exist, reset_model, msg_list, demisto_major_version, demisto_minor_version,
+                          model_data):
+    if debug:
+        if exist:
+            msg_list.append(MSG_MODEL_VERSION_IN_DEMISTO % (demisto_major_version, demisto_minor_version))
+        else:
+            msg_list.append(MSG_NO_MODEL_IN_DEMISTO)
+    if reset_model or not exist or (
+            demisto_major_version < MAJOR_VERSION and demisto_minor_version == MINOR_DEFAULT_VERSION):
+        msg_list.append(load_oob_model(OUT_OF_THE_BOX_MODEL_PATH))
+        model_64_str = get_model_data(URL_PHISHING_MODEL_NAME)[0]
+        model = decode_model_data(model_64_str)
+
+    elif demisto_major_version == MAJOR_VERSION:
+        model = decode_model_data(model_data)
+        msg_list.append(MSG_NO_ACTION_ON_MODEL)
+    elif (demisto_major_version < MAJOR_VERSION) and (demisto_minor_version > MINOR_DEFAULT_VERSION):
+        model_docker = load_model_from_docker()
+        model_docker_minor = model_docker.minor
+        model = load_demisto_model()
+        model_docker.logos_dict = model.logos_dict
+        model_docker.clf.named_steps.preprocessor.named_transformers_[
+            'image'].named_steps.trans.logo_dict = model.logos_dict
+        model_docker.minor += 1
+        save_model_in_demisto(model_docker)
+        msg_list.append(MSG_UPDATE_LOGO % (MAJOR_VERSION, model_docker_minor, model.major, model.minor))
+        model_64_str = get_model_data(URL_PHISHING_MODEL_NAME)[0]
+        model = decode_model_data(model_64_str)
+    else:
+        msg_list.append(MSG_WRONG_CONFIG_MODEL)
+        return_error(MSG_WRONG_CONFIG_MODEL)
+    return model, msg_list
+
+
+def main():
```

Comment:
Please add a `try` and `except` blocks here, and handle the errors being raised from the script inside the `except` block. If you're not sure exactly how to handle them, you can use other scripts / integrations in content as examples. `HelloWorld` can be a good example.